mods/lord/Blocks/remains/src/loot_functions.lua

```diff
@@ -25,13 +27,42 @@ local function get_random_items(count, items)
 	return result_items
 end
 
+--- Расчет гипотенузы (радиуса)
+--- c = math.sqrt(a^2+b^2)
+---
+--- @param a number                        right triangle catheter
+--- @param b number                        other right triangle catheter
+--- @return number                         calculated gipotenuse
+local function gipotenuse_calc(a,b)
+	local c = math_sqrt( a ^ 2 + b ^ 2 )
+
+	return c
+end
+
+--- Функция расчета точки на окружности, где центром является "лутоброс" (например останки рудокопа)
+--- radius - радиус от предмета до игрока
+--- angle угол смещения от игрока по окружности в радианах
+--- Для понимания:
+---    Ось Z(в minetest) это ось X на тригонометрическом круге
+---    Ось X(в minetest) это ось Y на тригонометрическом круге
+---    Угол - зеркальное отражение вектора player_look (куда смотрит игрок)
+--- @param radius number                   calculated by theoreme of Pifagor
+--- @param angle any                       mirror reflection of the player_look angle in radian
+local function around_drop(radius,angle)
+	local z_point, x_point
+	z_point = radius*cos(angle)
+	x_point = radius*sin(angle)
+
+	return x_point, z_point
```

Comment:
тут можно сразу вернуть результат, без дополнительных промежуточных переменных:
```suggestion
	return radius * sin(angle), radius * cos(angle)
```